libs/application/templates/marriage-conditions/src/lib/MarriageConditionsTemplate.ts

```diff
@@ -128,6 +131,7 @@ const MarriageConditionsTemplate: ApplicationTemplate<
                   Promise.resolve(val.done),
                 ),
               read: 'all',
+              delete: true,
```

Comment:
here, the applicant has already paid for the application, and assignee about to confirm. Does it make sense to allow to delete the application at that state?

Reply:
ill remove it